src/BookTable/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './style.scss';
+import image3 from '../image/image3.png';
+import image4 from '../image/image4.png';
+import divider4 from '../image/Divider4.png';
+import bookNow from '../image/Book now.png';
+
+function BookTable() {
+  return (
+    <section id="book" className="book_wrapper">
+      <div className="book_form">
+        <div className="left-side">
+          <img src={image3} alt="im3" />
+          <img src={image4} alt="im4" />
```

Comment:
alt should include string which has understandable text for person and screen readers. Pls change it

Reply:
i have changed it to more understandable text